docs/reference-guide/temporal_types/overview.md

```diff
@@ -0,0 +1,238 @@
+:-:|:-:
+year|Returns the year field|/
+month|Returns the month field|/
+day|Returns the day field|/
+
+Example:
+```cypher
+MATCH (a:Birthdays) RETURN f.Arnold.year
+```
+## LocalTime
+You can create `LocalTime` from a string or map by calling the function `localtime`. For strings, the local time format is specified by the ISO 8601:
+
+`[T]hh:mm:ss` or `[T]hh:mm` or `[T]hhmmss` or `[T]hhmm` or `[T]hh`
+
+Symbol table:
+
+name|description
+:-:|:-:
+|h|hours|
+|m|minutes|
+|s|seconds|
+
+`seconds` can be defined as decimal fractions with up to 6 digits. The first 3 digits represent milliseconds and the last 3 digits microseconds. For example the string `T22:10:32.300600` specifies `300` milliseconds and `600` microseconds.
+
+Example:
+```cypher
+CREATE (:School {Calculus : LocalTime("09:15:00")})
+```
+
+For maps there are 5 fields available: `hour`, `minute`, `second`, `millisecond` and `microsecond`.
+
+Example:
+```cypher
+CREATE (:School {Calculus : LocalTime({hour:9, minute:15})})
+```
+LocalTime properties:
+You can access the individual fields of LocalTime through their properties:
+
+name|description
+:-:|:-:
+hour|Returns the hour field|/
+minute|Returns the minute field|/
+second|Returns the second field|/
+millisecond|Returns the millisecond field|/
+microsecond|Returns the microsecond field|/
+
+Example:
+```cypher
+MATCH (s:School) RETURN f.Calculus.hour
+```
+## LocalDateTime
+You can create `LocalDateTime` from a string or map by calling the function `localdatetime`. For strings, the local time format is specified by the ISO 8601:
+
+`YYYY-MM-DDThh:mm:ss` or `YYYY-MM-DDThh:mm` or `YYYYMMDDThhmmss` or `YYYYMMDDThhmm` or `YYYYMMDDThh`
+
+Symbol table:
+
+name|description
+:-:|:-:
+|Y|Year|/
+|M|Month|/
+|D|Day|/
+|h|Hours|/
+|m|Minutes|/
+|s|Seconds|/
+
+Example:
+```cypher
+CREATE (:Flights {AIR123 : LocalDateTime("2021-10-05T14:15:00")})
+```
+For maps the following fields are available: `year`, `month`, `day`, `hour`, `minute`, `second`, `millisecond` and `microsecond`
+
+Example:
+```cypher
+CREATE (:Flights {AIR123 : LocalDateTime(year:2021, month:10, day:5, hour:14, minute:15)})
+```
+LocalDateTime properties:
+You can access the individual fields of LocalDateTime through its properties:
+
+name|description
+:-:|:-:
+year|Returns the year field|/
+month|Returns the month field|/
+day|Returns the day field|/
+hour|Returns the hour field|/
+minute|Returns the minute field|/
+second|Returns the second field|/
+millisecond|Returns the millisecond field|/
+microsecond|Returns the microsecond field|/
+
+Example:
+```cypher
+MATCH (f:Flights) RETURN f.AIR123.year
+```
+## Temporal types arithmetic
+
+Temporal types support native arithmetic and the operations are summarized in the following table:
+
+Duration operations: 
+
+op|result
+:-:|:-:
+duration + duration|duration|/
+duration - duration|duration|/
+-duration|negation|/
+
+Date operations: 
+
+op|result
+:-:|:-:
+date + duration|date|/
+duration + date|date|/
```

Comment:
Why the `duration + x` is only present for dates? If I am not mistaken this should either work for everything else or non of them.